src/server.ts

```diff
@@ -16,11 +17,23 @@ router.get("/nodes/:id", (ctx) => {
   ctx.body = getNodeAndChildren(Number(ctx.params.id))
 })
 
+router.post("/nodes", (ctx) => {
+  const body = ctx.request.body
+  ctx.body = storeNewNode(body.name, body.type, Number(body.parent))
+})
+
+router.del("/nodes", (ctx) => {
+  deleteNodes(ctx.request.body.ids)
+  ctx.body = "Deleted"
```

Comment:
status 204 means, "No Content". Maybe we don't need to return any text in the body?